python/src/aiconfig/editor/client/src/shared/prompt_schemas/DalleImageGenerationParserPromptSchema.ts

```diff
@@ -0,0 +1,35 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const DalleImageGenerationParserPromptSchema: PromptSchema = {
+  // "n", "quality", "response_format", "size", "style"
+
+  input: {
+    type: "string",
+  },
+  model_settings: {
+    type: "object",
+    properties: {
+      n: {
+        type: "integer",
+        minimum: 1,
+        maximum: 10,
```

Comment:
Just a note, Dall-E 3 only supports n=1. This we already flag as an error on the backend, but yea could be a P2 UX guide if we want to make it a bit more intuitive to users

<img width="630" alt="Screenshot 2024-01-02 at 22 32 12" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/d01387e4-3709-4c67-82c6-3f0b46eac51a">

I can't quite remember but I think the response just errors. I don't think it's a big deal, but worth testing just to sanity check that error propagates properly to UI. If not, please flag to me and I'll fix the model parser to raise this error explicitly in this situation!

Reply:
Neither dall-e model works for me -- I get, `Failed to run prompt: <class 'TypeError'>, Cannot instantiate typing.Union` for both. In this case, I was just ensuring the correct settings are propagated to the server from the client

Reply:
Ok I created a starter issue for this in https://github.com/lastmile-ai/aiconfig/issues/720. I will give to someone if they're interested in helping!